### PR TITLE
Fix: Refactored 404.tsx page to be consistent across all lfg-labs projects with provided references

### LINKED ISSUE #25: Make our `404.tsx` page consistent over our apps
## Description 📹

We want to make our `404.tsx` page consistent over our apps. This means we need to change [our 404.tsx page in this repo](https://github.com/lfglabs-dev/affiliate.starknet.id/blob/testnet/pages/404.tsx) to make it match [the app.starknet.id one](https://github.com/lfglabs-dev/app.starknet.id/blob/testnet/pages/404.tsx).

The only difference is that instead of using the `errorIllu.webp` you can use the `404illu.web` you'll find attached to this issue.

[404.zip](https://github.com/user-attachments/files/17764093/404.zip)

## Proposed Actions 🛠️

Here’s a checklist of actions to follow for resolving this issue:

1. **Fork and Create Branch**:  
   Fork the repository and create a new branch using the issue number:
   ```bash
   git checkout -b fix-[issue-number]
   ```

2. **Implement Changes**:  
- [ ] Refactor 404 page to make it similar [to this one](https://github.com/lfglabs-dev/app.starknet.id/blob/testnet/pages/404.tsx)
- [ ] Make sure the content is centered and give the same experience as here https://app.starknet.id/404example

3. **Run Tests and Commit Changes**:  
   Make sure your changes don't break existing functionality and commit with a clear message:
   ```bash
   git commit -m "Fix: [Short description of the fix]"
   ```

## Required 📋

To keep our workflow smooth, please make sure you follow these guidelines:

- **Assignment**: Don't create a pull request if you weren’t assigned to this issue.
- **Timeframe**: Complete the task within **3 business days**.
- **Closing the Issue**: In your PR description, close the issue by writing `Close #[issue_id]`.
- **Review Process**:
  - Once you've submitted your PR, change the label to **"ready for review"**.
  - If changes are requested, address them and then update the label back to **"ready for review"** once done.
- **Testing**: Test your PR locally before pushing, and verify that tests and build are working after pushing.

Thank you for your contribution 🙏

⚠️ WARNING: Failure to follow the requirements above may result in being added to the OnlyDust blacklist, affecting your ability to receive future rewards.

### PR DESCRIPTION
**Pull request by OWK50GA**
This PR addresses issue #25 by fixing the 404.tsx file of this repo to fit the one in the reference in the issue description. I've tested the changes locally and verified that the tests and build are working correctly.

Close #25

### CODE REVIEW

**Review by fricoben — CHANGES_REQUESTED** [review]
color of the text isn't consistent and the centering is also not take example here 

https://app.starknet.id/qsd

Capture d’écran 2024-11-23 à 21 31 07

**Comment by OWK50GA** [comment]
@fricoben the example link is not working.

**Comment by OWK50GA** [comment]
@fricoben the example link is not working 🙏🏽🙏🏽
I've made changes, but can't push yet because I've not confirmed with the example

File: pages/404.tsx [hunk 1]
```diff
@@ -5,10 +5,18 @@ import styles from "../styles/Home.module.css";
 const NotFound: NextPage = () => {
   return (
     <div className={styles.screen}>
-      <div className="flex items-center justify-center h-full">
-        <div className="text-center">
-          <h1 className="title">404 - Page Not Found</h1>
-          <p className="mt-2">The page you are looking for does not exist.</p>
+      <div className={styles.wrapperScreen}>
```
Comment (fricoben) [inline pages/404.tsx:8]: Capture d’écran 2024-11-24 à 21 31 38

You didn't test here, nothing has changed it's still not centered and in green.

is this class actually defined in `global.css`? The example I gave is another repo so you need to [see in this other repo how do we code it ](https://github.com/lfglabs-dev/app.starknet.id/blob/testnet/styles/globals.css)

**Review by fricoben — CHANGES_REQUESTED** [review]
Nothing changed here you didn't test anything, please fix

File: pages/404.tsx (hunk 1 above)
Comment (OWK50GA) [reply pages/404.tsx:8]: @fricoben the class is defined in global.css, yes. 
The thing is, it appears what is currently deployed is the brown-ish colour, but what is currently in the global.css for app.starknet.id is green

If the green is wrong, maybe you should open an issue for it to be worked on as well. 

For the centering, my bad it was an oversight. In the push I'm about to make I've changed it to the brown.

**Review by fricoben — APPROVED** [review]
Nope. I want it centered on the rose part not the red one.

Capture d’écran 2024-11-25 à 09 06 10

Why i

**Pull request merged**